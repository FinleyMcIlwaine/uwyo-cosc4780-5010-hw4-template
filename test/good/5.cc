// Macros to shut up the C++ warnings
#define printInt(x)
#define printDouble(x)
#define printString(x)
#define readInt() 0
#define readDouble() 0
#define readString() 0

int fib(int n)
{
    int res;
    if (n <= 2)
    {
        res = 1;
    }
    else
    {
        res = fib(n-1) + fib(n-2);
    }
    return res;
}

int main()
{
    printString("Which fibonacci num? ");
    int n = readInt();
    printInt(fib(n));
    return 0;
}
