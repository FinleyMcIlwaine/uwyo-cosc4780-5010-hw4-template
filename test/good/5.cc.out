Which fibonacci num? 6765
